x-pack/plugin/esql/qa/testFixtures/src/main/resources/null.csv-spec

```diff
@@ -0,0 +1,79 @@
+isNull
```

Comment:
Even though Coalesce can have an arbitrary number of parameters, the tests here always use two.

Reply:
`y=COALESCE(true, false, null)` doesn't seem to work. The value for `y` is `null`. Only if I the last value is non-null it works.

Reply:
Huh. I thought I had tests for all that in the unit tests. I'll add more.

Reply:
Interesting! the break here was with an optimizer rule. I'll push a fix.